### ISSUE TITLE
Homepage plugin: Remove accessibility violation filter and fix remaining a11y issues

### ISSUE THREAD

**Issue by christoph-jerolimov** [redhat-developer/rhdh-plugins#3587]
The homepage plugins has playwright tests that include a bunch of accessibility assertions.

The tests runs `runAccessibilityTests` from `@workspaces/homepage/e2e-tests/utils/accessibility.ts`, but some violations was ignored at the moment. The are filtered out via `getFilteredViolations`.

1. Remove the accessibility violation filter `getFilteredViolations` and its usage in `runAccessibilityTests`.
2. Run the playwright tests via `CI=1 yarn test:e2e` and list the left over issues.
3. Try to resolve these issues in the code base. When unsure, or not all issues can be removed, create a PR anyway.
